### ISSUE TITLE
Can't watch embedded videos

### ISSUE THREAD

**Issue by gabrc52** [lawrencehook/remove-youtube-suggestions#134]
I don't know if this is intentional or not but embedded videos always show as black for me:

![image](https://github.com/lawrencehook/remove-youtube-suggestions/assets/24363938/68fbc29e-1ccc-4400-ab12-8f2bc9351dfb)

I can't find the option to disable this, which I would like to do.

I am on Firefox 126.0 on Fedora Linux and RYS 4.3.59

**Comment by lawrencehook** [comment]
hmm works for me... do embedded videos still show black if you disable RYS entirely?

![image](https://github.com/lawrencehook/remove-youtube-suggestions/assets/6572030/1015309c-3f71-4816-a229-915a843e63d7)

**Comment by gabrc52** [comment]
No, the videos properly load with the extension disabled (whether from the extension settings or the browser itself).

**Comment by lawrencehook** [comment]
Can you send your export string? 

![image](https://github.com/lawrencehook/remove-youtube-suggestions/assets/6572030/21fbebec-08f2-4a7c-81c2-e0edc8d30bf8)

I cant try to recreate the bug using your settings.

**Comment by gabrc52** [comment]
These are my settings, thank you!

```
rys_settings_0:t,1:f,2:t,61:f,62:t,58:f,59:9<d1>00a-5<d1>00p,60:mo<d2>tu<d2>we<d2>th<d2>fr,54:t,55:t,74:t,10:f,56:t,3:t,4:t,5:t,6:f,7:f,8:f,9:f,11:f,79:f,12:f,13:f,14:f,15:f,16:f,17:f,18:f,19:f,20:f,63:f,64:f,21:t,22:f,23:f,24:f,69:f,25:f,78:f,26:f,28:f,29:t,30:f,31:f,76:f,80:f,73:f,33:f,27:f,75:f,32:f,34:f,35:f,36:f,37:f,38:f,39:f,40:f,41:t,42:f,43:f,44:f,65:f,72:f,77:f,68:f,45:f,46:f,47:f,48:f,71:f,49:f,50:f,51:f,52:t,67:f,57:f,66:f,70:f,81:f,53:f,82:f
```

**Comment by lawrencehook** [comment]
aha the culprit is "search engine mode"

I'll try to fix but in the meantime disabling that option should work.

**Comment by gabrc52** [comment]
I can confirm that disabling the option works. Thanks for diagnosing this so quickly!
